pathd/path_pcep_controller.h

```diff
@@ -75,22 +80,35 @@ struct pcep_ctrl_socket_data {
 typedef int (*pcep_ctrl_thread_callback)(struct thread *);
 
 /* Functions called from the main thread */
+int calculate_best_pce(struct ctrl_state *ctrl_state);
```

Comment:
All these functions are called only from the main thread ? if it is the case they should be names `pcep_ctrl_XXX`.
But from what I can see they are not and the prototype should be moved to the corresponding section.
Are all these functions used outside of `path_pcep_controller.c` ? if not the SHOULD NOT be there.

Reply:
changed